tests/e2e/backup_restore_suite_test.go

```diff
@@ -176,8 +176,7 @@ func runBackupAndRestore(brCase BackupRestoreCase, expectedErr error, updateLast
 
 	// uninstall app
 	log.Printf("Uninstalling app for case %s", brCase.Name)
-	err = UninstallApplication(dpaCR.Client, brCase.ApplicationTemplate)
-	Expect(err).ToNot(HaveOccurred())
+	UninstallApplication(dpaCR.Client, brCase.ApplicationTemplate)
```

Comment:
I do not like this, if we do not break if something goes wrong here, are we not changing the behavior of the test?

Reply:
The test is about backup and restore and not removal of namespace or resources using OpenShift go library/API. We of course should know why it failed to remove (finalizer??) that's why there is log, but imo failing each time finalizer is not done is something outside of oadp interest.

With my personal dev env I am using the following script to clean up OADP left overs, see that I have to clean finalizer on the restore object as it's always preventing that object to be properly removed. Maybe there is bug here preventing restore object to be properly removed, but I don't think it should stop every single PR from passing or introducing flakes.

```bash
#!/bin/bash

velero delete restore -n openshift-adp --all --confirm
velero delete backup -n openshift-adp --all --confirm

aws s3 rm s3://mprycoadp/ --recursive

# Patch restore objects with empty finalizer
for restore_object in $(oc get restore -n openshift-adp | grep -v NAME | awk '{print $1}'); do
    oc patch restore "$restore_object" -n openshift-adp --type merge --patch '
metadata:
  finalizers:
';
done

# Create an array of objects to be removed
ocp_object_types=("vsc" "backup" "backuprepository" "restore" "deletebackuprequest" "dataupload" "datadownload" "downloadrequest" "podvolumerestore")

for ocp_object in "${ocp_object_types[@]}"; do
    for ocp_object_instance in $(oc get "$ocp_object" -n openshift-adp |grep -v NAME|awk '{print $1}'); do
        oc delete "$ocp_object" "$ocp_object_instance" -n openshift-adp;
    done
done
```

Reply:
Restore does have finalizers, we have a test clean script https://github.com/openshift/oadp-operator/blob/master/Makefile#L516C1-L528C24

But I did not see problems with deleting applications and their namespaces in tests run yet (and if a problem occurs, when we restore, it can in the end doing nothing, as everything is still there. I think this IS NOT WHAT WE WANT)

I would hold this for discussion, at least check how many times this is erroring. I remember to see this in afterEach only once in all logs I saw

Reply:
On my system I see it 100% times after running e2e tests that's why I've added this to my local cleanup script.

Reply:
restores are not deleted after test suite run (for debuging), that's why we have the `make test-e2e-cleanup` command, to remove OADP stuff after a test. 

If you are seeing a problem with that script, that is another problem, not related to applications to backup/restore. 

I, for example, do not need to run `make test-e2e-cleanup` before running a test suite locally again, the leftovers from previous runs, do not interfere with the following run